roles/ceph-ansible/defaults/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# home directory which contains copy of files in ceph-ansible package
+home_dir: "{{ ansible_env['WORKSPACE'] | default(ansible_env['HOME']) }}"
```

Comment:
@mkudlej ah, yes you are right. Please, write there short description about that and consider the renaming of the ``home_dir`` to something like ``base_dir`` or ``work_dir``.